src/Tools/a2md/Program.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using System.Text;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace a2md
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            Loader loader = new Loader();
+            DescriptorEqualityComparer comparer = new DescriptorEqualityComparer();
+
+            var outputMarkdown = args
+                .Select(arg => new AnalyzerFileReference(arg, loader))
```

Comment:
Is there any way to get CodeFix info from an assembly? I'm think about modify this tool to also output if given warning has a corresponding CodeFix.